docs/code/MlNetHighLevelConcepts.md

```diff
@@ -21,6 +21,8 @@ This document is going to cover the following ML.NET concepts:
   - You can think of a machine learning *algorithm* as an estimator that learns on data and produces a machine learning *model* (which is a transformer).
 - [*Prediction function*](#prediction-function), represented as a `PredictionFunction<TSrc, TDst>` class.
   - The prediction function can be seen as a machine that applies a transformer to one 'row', such as at prediction time.
+- [*MLContext*](#mlcontext) object
+  - This is a 'catalog of everything' available in ML.NET. Use this object to read data, create estimators, save/load models, evaluate and perform all other tasks.
```

Comment:
Do we want a comment to create one per application?

Reply:
Nm, I see it's mentioned below.

> Create one `MLContext` for your entire application. All ML.NET components and operations are available as methods of various sub-objects of `MLContext`.

Perhaps we want a link to the below? `See more: [MLContext](#MLContext)` (assuming we can link to anchor tags in .md). Or just `See more below in section 'MLContext'`.

Reply:
Above is that very link: `[MLContext](#mlcontext)`

---
In reply to: [233236096](https://github.com/dotnet/machinelearning/pull/1612#discussion_r233236096) [](ancestors = 233236096)